config/routes.rb

```diff
@@ -49,6 +49,10 @@
   get "video" => "welcome#video"
   get "vote-for-us" => "welcome#nycbigapps"
 
+  namespace :admin do
+    resources :buildings
```

Comment:
Why namespace buildings under admin? Advocates will want to have access to buildings too, right? Or maybe I'm confusing admin with teammate again. Hmm. I think all the advocates are going to want access to building-wide data because it's pretty rare that advocates represent only one tenant in the building and not the others as well. Although I suppose if it did happen, we'd have painted ourselves in a corner by letting all advocates see buildings. Actually, no, they could just only see the tenants in the building they have access to, right? And what if we had an admin who was working with only some of the tenants in the building? Then we'd have the same problem. Hmm, somehow I get the feeling this wasn't the reason why you namespaced it under admin. I feel like we discussed this and I've simply forgotten. What was your thinking on that?